shared/middleware/helpers/redis_table.rb

```diff
@@ -47,6 +48,8 @@ def initialize(redis, pub_sub_api, shard_id, table_name, expire_in = nil)
   # @return [Hash] The inserted value, including the new :id field.
   def insert(value, ignored_ip=nil)
     new_id = next_id
+    new_uuid = SecureRandom.uuid
+    value = value.merge({'uuid' => new_uuid})
```

Comment:
It would take slightly more space in Redis (using the ID as both key and part of the value) and I suppose it introduces _some_ risk of inconsistency (getting a key that doesn't match the ID in its value).  We might add a little work to an update to make sure the ID can't be overwritten (in the same way the UUID is fixed right now). It would reduce work on reads and updates a little bit, removing the need to merge the ID into the returned row on demand.